src/amo/components/ThemeImage/styles.scss

```diff
@@ -2,8 +2,6 @@
 
 .ThemeImage {
   height: auto;
-  max-height: $theme-height-legacy;
-  max-width: $theme-width-default;
```

Comment:
The problem with removing this is that on 800px windows and larger we'll start stretching the image beyond its natural dimensions. On typical wide screens it will look really bad with themes containing a background image (the vast majority of themes do) since although the preview is an SVG, the background inside is JPG.

An example: http://127.0.0.1:3333/en-US/firefox/addon/a_theme_to_escalate/

It looks really pixelated on my wide monitor (ironically can't show that with a screenshot here as the limited width of the comment area in GitHub hides the problem by scaling it back down)

Reply:
I guess we are going to need some UX input. I'm not totally sure how to solve that problem other than max widthing the whole page. if you have the max width on the theme image then it looks weird as 2/3 the width of the card.